### ISSUE TITLE
Throttling Issue with ProviderError: Multiple Files to Unzip

### ISSUE THREAD

**Issue by Uriel1010** [morpheus65535/bazarr#2351]
I encountered a throttling issue while using Bazarr, specifically related to a ProviderError. The app indicates it is throttling "wizdom" for 10 minutes due to this error. 

Error Message
The exact error message displayed is:

```
Throttling wizdom for 10 minutes, until 24/01/08 13:50, because of: ProviderError. Exception info: "'More than one file to unzip' ~ wizdom.py@217"
```

it happend under to many shows and movies, while trying to download from wizdom provider (which is a hebrew subtitle provider)

**Comment by JaiZed** [comment]
This appears to be a provider issue.
I tried downloading 5 different subtitles from this provider for one specific movie.
I received exactly one subtitle in the zip file 3 of the times.
The other 2 times, the provider returned 2 subtitles within the zip file.
When I looked at the contents of the 2 files, one looked like Hebrew and the other was gibberish.

Subtitle File 1:
00:01:33,335 --> 00:01:35,921
".הרכבת למונטוק תצא מרציף בי"

Subtitle File 2:
00:01:33,335 --> 00:01:35,921
".äøëáú ìîåðèå÷ úöà îøöéó áé"


I'm guessing the second file does not have a UTF-8 encoding?
I'm also guessing it is Windows-1255.
How can we know which one to use?

**Comment by morpheus65535** [comment]
@JaiZed we guess based on language here: https://github.com/morpheus65535/bazarr/blob/master/libs/subliminal_patch/subtitle.py#L185-L186

**Comment by morpheus65535** [comment]
This one should be fixed in v1.4.1. Re-open if it isn't.
